webextensions/api/devtools.json

```
@@ -711,6 +1110,132 @@
               }
             }
           },
+          "openResource": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
```

First [appeared in the docs in Chrome 38](https://github.com/chromium/chromium/commit/8e78a4db865bb6c0256991209c760b432006cce6): 

```suggestion
                  "version_added": "≤38"
```